src/lib/homestore_backend/heap_chunk_selector.h

```diff
@@ -35,9 +35,10 @@ class HeapChunkSelector : public homestore::ChunkSelector {
 
     using VChunkHeap = std::priority_queue< VChunk, std::vector< VChunk >, VChunkComparator >;
     using VChunkDefragHeap = std::priority_queue< VChunk, std::vector< VChunk >, VChunkDefragComparator >;
+    using ChunkIdMap = std::unordered_map < homestore::chunk_num_t, homestore::chunk_num_t >;
```

Comment:
NIT: add a comment here to explain this is used for virtual chunk-> physical chunk map

Reply:
done